engine-tests/src/test/java/org/terasology/metatesting/BuildValidationTest.java

```diff
@@ -0,0 +1,55 @@
+package org.terasology.metatesting;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.net.URL;
+import java.util.Enumeration;
+
+public class BuildValidationTest {
+
+    @Test
+    public void testClasspathSanity() {
+        System.out.println("Running BuildValidationTest: Checking classpath sanity...");
+
+        // 1. Check if we can load a standard Java class
+        assertNotNull(String.class, "Should be able to load String class");
+
+        // 2. Check if we can find the TerasologyEngine class
+        URL resource = getClass().getResource("/org/terasology/engine/core/TerasologyEngine.class");
```

Comment:
I do not really like the idea of hard-coding paths to arbitrary class files here, even if it is just for a unit test. It makes the test fail if we ever re-name the class or change its package. Could we construct the class file path by using `TerasologyEngine.class.<something>` instead?

Reply:
Probably! I did notice this as odd vs easier and more generic things to test, but then that's the messy first pass largely from the one-handed-button-mashing-with-kids approach 👍